index.js

```diff
@@ -1,13 +1,13 @@
-import "intl";
-import "intl/locale-data/jsonp/en";
-import React from "react";
-import "./shim.js";
-import { AppRegistry } from "react-native";
-import WalletMigrate from "./screen/wallets/walletMigrate";
-import { name as appName } from "./app.json";
-import App from "./App";
-import LottieView from "lottie-react-native";
-import UnlockWith from "./UnlockWith.js";
+import 'intl';
+import 'intl/locale-data/jsonp/en';
+import React from 'react';
+import './shim.js';
+import { AppRegistry, StatusBar } from 'react-native';
+import WalletMigrate from './screen/wallets/walletMigrate';
+import { name as appName } from './app.json';
+import App from './App';
+import LottieView from 'lottie-react-native';
+import UnlockWith from './UnlockWith.js';
```

Comment:
would be worth to run prettier to fix all those formatting issues (so I wouldn't code review that ;) )